packages/aws-cdk-lib/cx-api/lib/features.ts

```diff
@@ -1088,6 +1089,20 @@ export const FLAGS: Record<string, FlagInfo> = {
     recommendedValue: true,
     compatibilityWithOldBehaviorMd: 'Set AWS::ECS::Service \'DeploymentAlarms\' manually to restore the previous behavior.',
   },
+
+  //////////////////////////////////////////////////////////////////////
+  [LOG_API_RESPONSE_DATA_PROPERTY_TRUE_DEFAULT]: {
```

Comment:
Does it make sense to add this feature flag? My thinking is that by remove `logApiResponseData` as a default event property we're now forcing another update for people who may not want it. For those people, they can enable this feature flag to maintain the existing behavior. For everyone else, they can keep it set to `false` and `logApiResponseData` will be undefined by default.

Reply:
I think the feature flag makes sense, but why is the recommended value `false`?

Reply:
`false` because it isn't the recommended behavior, just a convenience to anyone that has adopted it or adapted to it since CDK v2.138.0